arm/examples/check/check.go

```diff
@@ -31,7 +33,9 @@ func byInspecting() autorest.RespondDecorator {
 	}
 }
 
-func checkName(name string) {
+func main() {
+	name := fmt.Sprintf("testname%s", randomString(5))
```

Comment:
actually this random thing currently fails at the security scanner step we recently integrated. so it's just good to avoid this random in examples.